smac/optimizer/ei_optimization.py

```diff
@@ -21,24 +22,24 @@
 
 class AcquisitionFunctionMaximizer(object, metaclass=abc.ABCMeta):
     """Abstract class for acquisition maximization.
-    
+
     In order to use this class it has to be subclassed and the method
     ``_maximize`` must be implemented.
-    
+
     Parameters
     ----------
     acquisition_function : ~smac.optimizer.acquisition.AbstractAcquisitionFunction
-        
+
```

Comment:
I do not have a strong opinion on that, but what is the reason for these empty lines?

Reply:
This way PyCharm, fixing that